### PR TITLE
Add 🥈 Silver Financial Sponsors to the README

### PR DESCRIPTION
**Pull request by Will5**
## ☕️ Reasoning

This pull request adds 🥈 Silver Financial Sponsors to the README 🎉

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged

### CODE REVIEW

File: packages/next-auth/README.md [hunk 1]
```diff
@@ -189,6 +189,20 @@ We're happy to announce we've recently created an [OpenCollective](https://openc
...
     <tr>
+      <td align="center" valign="top">
+        <a href="https://vercel.com" target="_blank">
+          <img width="128px" src="https://avatars.githubusercontent.com/u/69983493?s=200&v=4" alt="Stytch Logo" />
+        </a><br />
+        <div>Stytch</div><br />
+        <sub>🥈 Silver Financial Sponsor</sub>
+      </td>
+      <td align="center" valign="top">
+        <a href="https://vercel.com" target="_blank">
```
Comment (ThangHuuVu) [inline packages/next-auth/README.md:200]: This should link to Beyond Identity

File: packages/next-auth/README.md [hunk 2]
```diff
@@ -189,6 +189,20 @@ We're happy to announce we've recently created an [OpenCollective](https://openc
 <table>
   <tbody>
     <tr>
+      <td align="center" valign="top">
+        <a href="https://vercel.com" target="_blank">
```
Comment (ThangHuuVu) [inline packages/next-auth/README.md:193]: This should link to Stytch

**Review by ThangHuuVu — CHANGES_REQUESTED** [review]

File: packages/next-auth/README.md (hunk 2 above)
Comment (Will5) [reply packages/next-auth/README.md]: Good catch! Thanks!

File: packages/next-auth/README.md (hunk 1 above)
Comment (Will5) [reply packages/next-auth/README.md]: Good catch! Thanks!

**Review by ThangHuuVu — APPROVED** [review]

**Comment by ThangHuuVu** [comment]
@Will5 LGTM, please update the branch before we merge 🙌 I'm not sure why I can't update your PR on GitHub UI, normally I can do that by myself 😅

**Pull request merged**